File: SystemERP - ASP.NET/ErpServicesASP.API/Dto/EmpresaCreateDto.cs
using ErpServicesASP.API.Model;

namespace ErpServicesASP.API.Dto
{
    public class EmpresaCreateDto
    {
        public string Name { get; set; }
        public string NomeFantasia { get; set; }
        public int idDono { get; set; }
        public string CNPJ { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
        public string Endereco { get; set; }
        public int idTipoDeEmpresa { get; set; }
        public DateTime DataCriacao { get; set; }
    }
}

File: SystemERP - React/models/EmpresaCreateDto.ts
export interface EmpresaCreateDto {
    name: string
    nomeFantasia?: string
    idDono?: number
    cnpj: string
    email: string
    setor?: string
    telefone?: string
    endereco?: string
    idTipoDeEmpresa?: number
    dataCriacao?: string
  }